odl/contrib/fom/test/test_supervised.py

```diff
@@ -68,5 +68,62 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_mean_value_difference_sign():
```

Comment:
Use a fixture for space, as in the ``test_mean_value_difference_range_value``, or is this not necessary? Could tolerance be calibrated give what space one tests against, or is this over-kill?